openverse_catalog/dags/providers/provider_dag_factory.py

```diff
@@ -75,6 +79,7 @@
 OPENVERSE_BUCKET = os.getenv("OPENVERSE_BUCKET")
 OUTPUT_DIR_PATH = os.path.realpath(os.getenv("OUTPUT_DIR", "/tmp/"))
 DATE_RANGE_ARG_TEMPLATE = "{{{{ macros.ds_add(ds, -{}) }}}}"
+DATE_PARTITION_ARG_TEMPLATE = "{}/{}/{{{{ date_partition_for_prefix(dag.schedule_interval, dag_run.logical_date) }}}}"  # noqa
```

Comment:
Can the first two holes have names, so it's clear in the template what they're for?

```suggestion
DATE_PARTITION_ARG_TEMPLATE = "{media_type}/{provider_type}/{{{{ date_partition_for_prefix(dag.schedule_interval, dag_run.logical_date) }}}}"  # noqa
```